<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>huesaturation</class>
 <widget class="QWidget" name="huesaturation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QSlider" name="HueSlider">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>180</y>
     <width>251</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>40</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Контраст</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>150</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Цветовые оттенки</string>
   </property>
  </widget>
  <widget class="QSlider" name="SaturationSlider">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>60</y>
     <width>251</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="AcceptButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>260</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Принять</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CancelButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Отмена</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>80</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>200</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>80</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Значение</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>200</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Значение</string>
   </property>
  </widget>
  <widget class="QPushButton" name="AutoButton">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>260</y>
     <width>83</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Авто</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
